### DISCUSSION TITLE
Sous réseau

### DISCUSSION THREAD

**Discussion by llagar910e** [dbarzin/mercator#999] [Ideas]
Bonjour,

Nous sommes éditeur de logiciel et nous administrons des serveurs de nos clients. 
Nous sommes emmenés à nous connecter à de nombreux réseaux locaux au travers de VPN, certains pouvant avoir le même range d'adresses IP. 
Je pense qu'il serait intéressant de pouvoir sélectionner le ou les réseau/sous réseau dans lequel se trouve un serveur, poste de travail, etc.
Actuellement ça semble calculé par rapport à l'IP.

Merci

**Comment by dbarzin** [comment]
Traité dans https://github.com/dbarzin/mercator/issues/1001
